docs/docs/user-guide/users.md

```diff
@@ -1,133 +1,52 @@
-- **Editors** can edit all data in the schema, but cannot change the underlying data structures or manage access.
-- **Viewers** have read-only access to all data in the schema. They cannot make any changes.
+Admin users have the following capabilities which Standard users do not:
 
-### Manager
+- Admins can can manage other Mathesar users (view, add, edit, delete).
+- Admins can add and remove [Databases](./databases.md).
+- Admins can manage [Collaborators](./permissions.md#collaborators). This allows an Admin user to grant any Mathesar user access to a database through a PostgreSQL role that the Admin specifies.
 
-- Can view, add, and remove other users' access to the schema.
-- Can view, add, edit, and remove any schema in the schema.
-- Can view, add, edit, and remove any table in the schema.
-- Can view, add, edit, and remove any column in the schema.
-- Can view, add, edit, and remove any constraint in the schema.
-- Can view, add, edit, and remove any record in the schema.
-- Can view, add, edit, and remove any exploration in the schema.
-- Can perform "extract column" and "move column" actions
+Upon installing Mathesar, your first user will be an Admin user.
 
-### Editor
+## Users vs Roles
 
-- Can view any table in the schema.
-- Can view any column in the schema.
-- Can view any constraint in the schema.
-- Can view, add, edit, and remove any record in the schema.
-- Can view, add, edit, and remove any exploration in the schema.
+- A **"user"** is a Mathesar construct. Each Mathesar installation has multiple users.
+- A **"role"** is a PostgreSQL construct ([docs](https://www.postgresql.org/docs/current/user-manag.html)). Each PostgreSQL server has multiple roles and multiple databases.
 
-### Viewer
+!!! caution "Why this distinction is important"
+    Outside of Mathesar, it's not uncommon for people to say _user_ when referring to a PostgreSQL _role_. However, within the context of Mathesar users and roles are different things! Our documentation maintains this distinction pedantically.
 
-- Can view any table in the schema.
-- Can view any column in the schema.
-- Can view any constraint in the schema.
-- Can view any exploration in the schema.
-- Can use the **Data Explorer** to create ad-hoc explorations using data in the schema, but cannot save them.
+How users and roles work together:
 
-## Managing Schema Roles
+- To access a database, each Mathesar user must be assigned a PostgreSQL role to be used for that database.
+- The user's permissions on anything _inside_ the database are determined by the corresponding role's permissions within PostgreSQL.
 
-!!! note
-    - Only **Admins**, **Database Managers**, and **Schema Managers** can manage access to a schema.
+    !!! info "Admin doesn't matter here"
+        The user's "admin" status with Mathesar _has no effect_ on the user's ability to do things within a database! It only affects operations outside the database.
```

Comment:
Can we add an extra clarification on "outside the database"?

The admin can perform operations like adding collaborators and modifying Role Configurations, which are technically outside the database, but the UI is inside the concept of the Database we present to the user.

Reply:
I've updated this to add a reference to role configuration and collaborator management.